app/mailers/curation_mailer.rb

```diff
@@ -33,4 +33,16 @@ def events_require_approval(provider, cut_off_time)
       format.text
     end
   end
+
+  def materials_require_approval(provider, cut_off_time)
+    @provider = provider
+    return unless @provider.event_curation_email.present?
```

Comment:
Lets rename this field to `content_curation_email` now that is not just for events.